### ISSUE TITLE
Preset modes "not sticky" to A/C modes

### ISSUE THREAD

**Issue by IGaMbLeRI** [georgezhao2010/midea_ac_lan#42]
Thank you for the update.

My units support "ECO" function in Cooling mode only. The units also do not have "Comfort" function. However, now whenever the units are switched on, they choose "Comfort" preset. I activate "Eco" preset, but it switches to "Comfort" after 2 seconds, although ECO function on A/C gets activated.

When I change mode to anything else, it just chooses "Comfort", even if I put "None" - it still switches to "Comfort", like Dry - Comfort. It even changes to "Comfort" itself from "Eco" for some reason within the same A/C and mode.

I expect the preset to apply to specific A/C modes, for example, I have Cooling ON, I go to Presets and select ECO, from now on, this A/C will switch to ECO function every time the A/C is in Cooling mode, while other modes should have their own Preset. Or not switch away from "None" which applies to all modes. Impossible to have ECO on all modes, at least on my model.

**Comment by IGaMbLeRI** [comment]
Here is Comfort function toggling by itself (no automations or manual switching).

```
2022-08-02 04:29:28 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 25.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:29:29 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749179] Received: {'header': 'aa22ac00000000000303', 'body': 'c00068667f7f00300000006469000000000000000000c854', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:29:29 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 25.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:29:35 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e198', 'message type': '03', 'body type': '41'}
2022-08-02 04:29:36 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.devices.ac.device] [16492674483964] Received: {'header': 'aa22ac00000000000303', 'body': 'c00049667f7f00300000005f69000000000000000000679c', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:29:36 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Status update: {'power': False, 'mode': 2, 'target_temperature': 25.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:29:40 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e324', 'message type': '03', 'body type': '41'}
2022-08-02 04:29:41 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749164] Received: {'header': 'aa22ac00000000000303', 'body': 'c00048667f7f00300000005e68000000000000000000d9e6', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:29:41 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Status update: {'power': False, 'mode': 2, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:30:30 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e5f9', 'message type': '03', 'body type': '41'}
2022-08-02 04:30:31 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749179] Received: {'header': 'aa22ac00000000000303', 'body': 'c00068667f7f0030000000636900000000000000000002e1', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:30:31 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 24.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:30:37 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e745', 'message type': '03', 'body type': '41'}
2022-08-02 04:30:38 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.devices.ac.device] [16492674483964] Received: {'header': 'aa22ac00000000000303', 'body': 'c00049667f7f00300000005f69000000000000000000b429', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:30:38 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Status update: {'power': False, 'mode': 2, 'target_temperature': 25.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:30:42 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e95a', 'message type': '03', 'body type': '41'}
2022-08-02 04:30:43 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749164] Received: {'header': 'aa22ac00000000000303', 'body': 'c00048667f7f00300000005f680000000000000000009da9', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:30:43 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Status update: {'power': False, 'mode': 2, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:31:31 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000ebe6', 'message type': '03', 'body type': '41'}
2022-08-02 04:31:33 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749179] Received: {'header': 'aa22ac00000000000303', 'body': 'c00068667f7f003000000063690000000000000000000f1c', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:31:33 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 24.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:31:38 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000ed3b', 'message type': '03', 'body type': '41'}
2022-08-02 04:31:39 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.devices.ac.device] [16492674483964] Received: {'header': 'aa22ac00000000000303', 'body': 'c00049667f7f00300000005f69000000000000000000b88a', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:31:39 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Status update: {'power': False, 'mode': 2, 'target_temperature': 25.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:31:44 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000ef87', 'message type': '03', 'body type': '41'}
2022-08-02 04:31:45 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749164] Received: {'header': 'aa22ac00000000000303', 'body': 'c00048667f7f00300000005f680000000000000000006a5e', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:31:45 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Status update: {'power': False, 'mode': 2, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:32:34 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000f105', 'message type': '03', 'body type': '41'}
2022-08-02 04:32:34 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749179] Received: {'header': 'aa22ac00000000000303', 'body': 'c00068667f7f00300000006369000000000000000000eaca', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:32:34 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 24.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:32:39 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000f3b9', 'message type': '03', 'body type': '41'}
2022-08-02 04:32:40 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.devices.ac.device] [16492674483964] Received: {'header': 'aa22ac00000000000303', 'body': 'c00049667f7f00300000005f6900000000000000000034a5', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:32:40 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Status update: {'power': False, 'mode': 2, 'target_temperature': 25.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
2022-08-02 04:32:45 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000f564', 'message type': '03', 'body type': '41'}
2022-08-02 04:32:46 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749164] Received: {'header': 'aa22ac00000000000303', 'body': 'c00048667f7f00300000005f68000000000000000000874a', 'message type': '03', 'body type': 'c0'}
2022-08-02 04:32:46 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Status update: {'power': False, 'mode': 2, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
```

**Comment by georgezhao2010** [comment]
> Thank you for the update.
> 
> My units support "ECO" function in Cooling mode only. The units also do not have "Comfort" function. However, now whenever the units are switched on, they choose "Comfort" preset. I activate "Eco" preset, but it switches to "Comfort" after 2 seconds, although ECO function on A/C gets activated.
> 
> When I change mode to anything else, it just chooses "Comfort", even if I put "None" - it still switches to "Comfort", like Dry - Comfort. It even changes to "Comfort" itself from "Eco" for some reason within the same A/C and mode.
> 
> I expect the preset to apply to specific A/C modes, for example, I have Cooling ON, I go to Presets and select ECO, from now on, this A/C will switch to ECO function every time the A/C is in Cooling mode, while other modes should have their own Preset. Or not switch away from "None" which applies to all modes. Impossible to have ECO on all modes, at least on my model.

I got it. 

`c0014b667f7f00000004006e780f009300000000000001f6b8` , It's my 'c0' message body.
`c00048667f7f00300000005f68000000000000000000874a`, It's yours.

See? it is one byte less than mine, it caused this issue.

Anyway, Thank you for reporting, I'll fix it in next version.

**Comment by georgezhao2010** [comment]
> Here is Comfort function toggling by itself (no automations or manual switching).
> 
> ```
> 2022-08-02 04:29:28 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 25.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:29:29 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749179] Received: {'header': 'aa22ac00000000000303', 'body': 'c00068667f7f00300000006469000000000000000000c854', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:29:29 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 25.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:29:35 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e198', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:29:36 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.devices.ac.device] [16492674483964] Received: {'header': 'aa22ac00000000000303', 'body': 'c00049667f7f00300000005f69000000000000000000679c', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:29:36 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Status update: {'power': False, 'mode': 2, 'target_temperature': 25.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:29:40 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e324', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:29:41 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749164] Received: {'header': 'aa22ac00000000000303', 'body': 'c00048667f7f00300000005e68000000000000000000d9e6', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:29:41 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Status update: {'power': False, 'mode': 2, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:30:30 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e5f9', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:30:31 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749179] Received: {'header': 'aa22ac00000000000303', 'body': 'c00068667f7f0030000000636900000000000000000002e1', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:30:31 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 24.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:30:37 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e745', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:30:38 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.devices.ac.device] [16492674483964] Received: {'header': 'aa22ac00000000000303', 'body': 'c00049667f7f00300000005f69000000000000000000b429', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:30:38 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Status update: {'power': False, 'mode': 2, 'target_temperature': 25.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:30:42 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000e95a', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:30:43 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749164] Received: {'header': 'aa22ac00000000000303', 'body': 'c00048667f7f00300000005f680000000000000000009da9', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:30:43 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Status update: {'power': False, 'mode': 2, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:31:31 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000ebe6', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:31:33 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749179] Received: {'header': 'aa22ac00000000000303', 'body': 'c00068667f7f003000000063690000000000000000000f1c', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:31:33 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 24.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:31:38 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000ed3b', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:31:39 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.devices.ac.device] [16492674483964] Received: {'header': 'aa22ac00000000000303', 'body': 'c00049667f7f00300000005f69000000000000000000b88a', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:31:39 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Status update: {'power': False, 'mode': 2, 'target_temperature': 25.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:31:44 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000ef87', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:31:45 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749164] Received: {'header': 'aa22ac00000000000303', 'body': 'c00048667f7f00300000005f680000000000000000006a5e', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:31:45 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Status update: {'power': False, 'mode': 2, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:32:34 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000f105', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:32:34 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749179] Received: {'header': 'aa22ac00000000000303', 'body': 'c00068667f7f00300000006369000000000000000000eaca', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:32:34 DEBUG (Thread-3) [custom_components.midea_ac_lan.midea.core.device] [18691697749179] Status update: {'power': False, 'mode': 3, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 24.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:32:39 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000f3b9', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:32:40 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.devices.ac.device] [16492674483964] Received: {'header': 'aa22ac00000000000303', 'body': 'c00049667f7f00300000005f6900000000000000000034a5', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:32:40 DEBUG (Thread-5) [custom_components.midea_ac_lan.midea.core.device] [16492674483964] Status update: {'power': False, 'mode': 2, 'target_temperature': 25.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': False, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> 2022-08-02 04:32:45 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Sending: {'header': 'aa20ac8c000000000003', 'body': '418100ff03ff0002000000000000000000000000f564', 'message type': '03', 'body type': '41'}
> 2022-08-02 04:32:46 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.devices.ac.device] [18691697749164] Received: {'header': 'aa22ac00000000000303', 'body': 'c00048667f7f00300000005f68000000000000000000874a', 'message type': '03', 'body type': 'c0'}
> 2022-08-02 04:32:46 DEBUG (Thread-4) [custom_components.midea_ac_lan.midea.core.device] [18691697749164] Status update: {'power': False, 'mode': 2, 'target_temperature': 24.0, 'fan_speed': 102, 'swing_vertical': False, 'swing_horizontal': False, 'boost_mode': False, 'smart_eye': False, 'dry': False, 'eco_mode': False, 'aux_heat': False, 'night_light': False, 'natural_wind': False, 'temp_fahrenheit': False, 'screen_display': False, 'comfort_mode': True, 'indoor_temperature': 22.0, 'outdoor_temperature': 27.0, 'indirect_wind': False, 'available': True}
> ```

Is it solved in v0.3.3? If yes please close this issue.
